[2024-03-29T17:00:55.623+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo_scheduling.hello scheduled__2022-03-08T00:00:00+00:00 [queued]>
[2024-03-29T17:00:55.666+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo_scheduling.hello scheduled__2022-03-08T00:00:00+00:00 [queued]>
[2024-03-29T17:00:55.670+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-03-29T17:00:55.701+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): hello> on 2022-03-08 00:00:00+00:00
[2024-03-29T17:00:55.710+0000] {standard_task_runner.py:60} INFO - Started process 2977 to run task
[2024-03-29T17:00:55.729+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'demo_scheduling', 'hello', 'scheduled__2022-03-08T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/demo_scheduling.py', '--cfg-path', '/tmp/tmp5sfbn0cd']
[2024-03-29T17:00:55.743+0000] {standard_task_runner.py:88} INFO - Job 32: Subtask hello
[2024-03-29T17:00:55.798+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-29T17:00:55.924+0000] {task_command.py:423} INFO - Running <TaskInstance: demo_scheduling.hello scheduled__2022-03-08T00:00:00+00:00 [running]> on host c4cb237119ff
[2024-03-29T17:00:56.220+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='domwatcharin' AIRFLOW_CTX_DAG_ID='demo_scheduling' AIRFLOW_CTX_TASK_ID='hello' AIRFLOW_CTX_EXECUTION_DATE='2022-03-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-08T00:00:00+00:00'
[2024-03-29T17:00:56.224+0000] {demo_scheduling.py:9} INFO - Hello
[2024-03-29T17:00:56.225+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-03-29T17:00:56.243+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=demo_scheduling, task_id=hello, execution_date=20220308T000000, start_date=20240329T170055, end_date=20240329T170056
[2024-03-29T17:00:56.313+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-29T17:00:56.365+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
